apps/consent/package.json

```diff
@@ -32,13 +32,14 @@
     "edge-csrf": "^1.0.6",
     "graphql": "^16.8.1",
     "libphonenumber-js": "^1.10.51",
-    "next": "14.0.4",
+    "next": "14.0.1",
```

Comment:
why downgrade?

Reply:
Cypress tests were failing with this version; they were flaky. I will look into this, but I think that would be a topic for another PR.